cob_bringup/package.xml

```diff
@@ -12,72 +12,70 @@
   <author email="fmw@ipa.fhg.de">Florian Weisshardt</author>
 
   <buildtool_depend>catkin</buildtool_depend>
- 
   <build_depend>roslaunch</build_depend>
- 
-  <run_depend>cob_undercarriage_ctrl</run_depend>
+
+  <run_depend>camera1394</run_depend>
+  <run_depend>canopen_motor_node</run_depend>
   <run_depend>cob_base_drive_chain</run_depend>
-  <run_depend>cob_lbr</run_depend>
-  <run_depend>cob_head_axis</run_depend>
-  <run_depend>cob_sound</run_depend>
-  <run_depend>cob_light</run_depend>
-  <run_depend>schunk_sdh</run_depend>
-  <run_depend>schunk_powercube_chain</run_depend>
-  <run_depend>robot_state_publisher</run_depend>
-  <run_depend>ur_driver</run_depend>
-  <run_depend>cob_relayboard</run_depend>
-  <run_depend>cob_voltage_control</run_depend>
-  <run_depend>cob_safety_controller</run_depend>
-  <run_depend>cob_sick_s300</run_depend>
-  <run_depend>cob_sick_lms1xx</run_depend>
-  <run_depend>cob_camera_sensors</run_depend>
-  <run_depend>openni_launch</run_depend>
-  <run_depend>openni2_launch</run_depend>
+  <run_depend>cob_base_velocity_smoother</run_depend>
+  <run_depend>cob_calibration_data</run_depend>
   <run_depend>cob_cam3d_throttle</run_depend>
-  <run_depend>image_proc</run_depend>
-  <run_depend>hokuyo_node</run_depend>
-  <run_depend>prosilica_camera</run_depend>
-  <!--run_depend>camera1394</run_depend-->
-  <run_depend>nodelet</run_depend>
+  <run_depend>cob_camera_sensors</run_depend>
+  <run_depend>cob_collision_velocity_filter</run_depend>
   <run_depend>cob_command_gui</run_depend>
+  <run_depend>cob_control_mode_adapter</run_depend>
   <run_depend>cob_dashboard</run_depend>
-  <run_depend>cob_teleop</run_depend>
-  <run_depend>cob_collision_velocity_filter</run_depend>
+  <run_depend>cob_default_env_config</run_depend>
+  <run_depend>cob_default_robot_config</run_depend>
   <run_depend>cob_footprint_observer</run_depend>
-  <run_depend>cob_base_velocity_smoother</run_depend>
-  <run_depend>cob_monitoring</run_depend>
-  <run_depend>joy</run_depend>
-  <run_depend>rviz</run_depend>
-  <run_depend>diagnostic_aggregator</run_depend>
-  <run_depend>topic_tools</run_depend>
-  <run_depend>costmap_2d</run_depend>
+  <run_depend>cob_frame_tracker</run_depend>
   <run_depend>cob_hardware_config</run_depend>
-  <run_depend>cob_default_robot_config</run_depend>
-  <run_depend>cob_default_env_config</run_depend>
-  <run_depend>cob_phidgets</run_depend>
-  <run_depend>prace_gripper_driver</run_depend>
-  <run_depend>frida_driver</run_depend>
+  <run_depend>cob_head_axis</run_depend>
   <run_depend>cob_image_flip</run_depend>
-  <run_depend>schunk_sdhx</run_depend>
+  <run_depend>cob_lbr</run_depend>
```

Comment:
Well, there is still the launch file https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/drivers/lbr_driver.launch